lib/kdump_utils.pm

```diff
@@ -110,7 +110,7 @@ sub kdump_is_active {
             record_soft_failure 'bsc#1021484 -- fail to create kdump initrd';
             assert_script_run 'systemctl restart kdump';
             $status = script_output('systemctl status kdump ||:');
```

Comment:
I guess we can get rid of this line also then. I'm also not sure if we want to see 10 soft-failures, but just one, and keep restarting it.

Reply:
@rwx788 not possible. 

```assert_script_run 'systemctl restart kdump';``` - causes hard fail of test if restart of service fails